changelog/mangled_func_kw.dd

```diff
@@ -0,0 +1,16 @@
+Added `__MANGLED_FUNCTION__` special keywords
+
+This special keyword will allow the usage of the mangled function symbol on
+situations where `mixin(callee).mangleof` don't work.
```

Comment:
This should have an example of when this is useful in a real project.
Saying it's for situations "where `mixin(callee).mangleof` doesn't work" just raises the question "when would I use `mixin(callee).mangleof`".